[en]update virtnest-vm docs (#5142)
DaoCloud/DaoCloud-docs: docs/en/docs/virtnest/vm/vm-network.md

1. When selecting the Bridge network mode, some information needs to be configured in advance:

    - Install and run Open vSwitch on the host nodes. See  [Ovs-cni Quick Start](https://spidernet-io.github.io/spiderpool/v0.9/usage/install/underlay/get-started-ovs/#_1).
    - Configure Open vSwitch bridge on the host nodes. See [Vswitch](https://spidernet-io.github.io/spiderpool/v0.9/usage/install/underlay/get-started-ovs/#configure-open-vswitch-bridge-on-the-node) for instructions.

>     - Install Spiderpool. See [Installing Spiderpool](https://docs.daocloud.io/en/network/modules/spiderpool/install/install/#how-to-install-spiderpool) for instructions. By default, Spiderpool will install both Multus CNI and Ovs CNI.

Reviewer: ```suggestion
    - Install Spiderpool. See [Installing Spiderpool](../../../network/modules/spiderpool/install/install.md#how-to-install-spiderpool) for instructions. By default, Spiderpool will install both Multus CNI and Ovs CNI.
```
需要用相对路径

        ![Network Mode](../images/createvm-net02.png)
        

>         - When selecting Bridge mode, there are some [prerequisites](#prerequisites).

Reviewer: ```suggestion
        - Ensure all [prerequisites](#prerequisites) are met before selecting the Bridge mode.
```
